packages/editor/templates/Coin/README.md

```diff
@@ -236,6 +236,15 @@ function isValidAmount(amount) {
 
 And that's all the code. To see the dapp in action, click _Compile_, then _Deploy_ found under the disclosure triangle of the contract file, then open the _Preview_ tab to see the frontend of the dapp.
 
-## Find out more
+## Next Steps
 
 You can read a full tutorial that accompanies this example dapp, plus many more tutorials, on [kauri.io](https://kauri.io/article/bdd65d6155a74b8aa52672b46b7230a8/v1/a-fullstack-dapp-for-creating-tokens).
+
+Other resources useful to continue your dapp development journey are:
+
+- [Remix](https://remix.ethereum.org/): Ethereum IDE and tools for the web
+- [Grid](https://grid.ethereum.org/): Download, configure, and run Ethereum nodes and tools
+- [CryptoZombies](https://cryptozombies.io/): Learn Solidity building your own Zombie game
```

Comment:
I'd move CryptoZombies to the top of this list. From the developers I've talked to they all said this is the top resource they recommend to new people in the space looking to learn Solidity.